lib/code_generator/generator.rb

```diff
@@ -170,6 +171,62 @@ def #{m_name}(#{arguments})
       end
     end
 
+    def generate_public_private_methods
```

Comment:
There was a mistake from my side. I mixed up method names in a hurry. We should add a generator(s) for private methods and private class methods, not for public private methods (it's impossible anyway in Ruby and confuses all of OOP rules). I found and corrected typo in README. My bad... Correct TODO is: https://github.com/unurgunite/code_generator/tree/development?tab=readme-ov-file#todo